docs/source/contributor-guide/debugging.md

```diff
@@ -20,12 +20,13 @@ under the License.
 # Comet Debugging Guide
 
 This HOWTO describes how to debug JVM code and Native code concurrently. The guide assumes you have:
+
 1. Intellij as the Java IDE
 2. CLion as the Native IDE. For Rust code, the CLion Rust language plugin is required. Note that the
```

Comment:
I feel that we can move to RustRover instead of CLion. It is specified for Rust development. We can update this later.